index.html

```diff
@@ -64,6 +64,36 @@
         </gu-column>
       </gu-row>
 
+      <h2>Boxes</h2>
+      <gu-row>
+        <gu-column span="4">
+          <gu-box variant="primary">
+            This is some content
+          </gu-box>
+        </gu-column>
+        <gu-column span="4" gu-box="secondary">
```

Comment:
Depends on how you see it! In the attribute case, you're putting things into the column. Anyway, might just be me, but the word "box" made me think that box had some more semantic meaning about grouping things together rather than be about styling.